src/openrct2-ui/windows/Banner.cpp

```diff
@@ -133,10 +133,9 @@ rct_window* window_banner_open(rct_windownumber number)
     window_init_scroll_widgets(w);
 
     auto banner = GetBanner(w->number);
-    int32_t view_x = banner->position.x << 5;
-    int32_t view_y = banner->position.y << 5;
+    auto bannerViewPos = banner->position.ToCoordsXY();
```

Comment:
```suggestion
    auto bannerViewPos = banner->position.ToCoordsXY().ToTileStart();
```
and remove the += at line 154.

Reply:
Can't this theoretically return a different `TileElement*`? Because the sum happens after the `map_get_first_element_at`

Reply:
No. Both represent the same tile, and the structure holding them divides the x and y by 32 anyway.